src/components/NewWorkspaceModal.js

```diff
@@ -98,14 +100,18 @@ export default class NewWorkspaceModal extends Component {
       'membersGroupName',
       cloneWorkspace && cloneWorkspace.workspace.authorizationDomain
     )
-    const errors = validate({ namespace, name }, constraints, { fullMessages: false })
+    const errors = validate({ namespace, name }, constraints, {
+      prettify: v => ({ namespace: 'Billing project', name: 'Name' }[v] || validate.prettify(v))
+    })
     return h(Modal, {
       title: cloneWorkspace ? 'Clone a Workspace' : 'Create a New Workspace',
       onDismiss,
-      okButton: buttonPrimary({
-        disabled: errors,
-        onClick: () => this.create()
-      }, cloneWorkspace ? 'Clone Workspace' : 'Create Workspace')
+      okButton: h(TooltipTrigger, { content: Utils.summarizeErrors(errors) }, [
```

Comment:
I had to look it up, I feel like we should mention somewhere like in the `TooltipTrigger` docstring that when `content` is falsy we don't show an empty tooltip.

Reply:
Done.